test/test_node_application.py

```diff
@@ -21,6 +21,11 @@ def test_node_application_basic():
 
     assert_basic_application()
 
+def test_node_application_options():
+    client.load('options')
+
+    assert_basic_application()
+
```

Comment:
I would suggest to check (in couple tests) that we don't have new error message in `unit.log`. Currently it should fail:

```
diff --git a/test/test_node_application.py b/test/test_node_application.py
--- a/test/test_node_application.py
+++ b/test/test_node_application.py
@@ -10,33 +10,36 @@ prerequisites = {'modules': {'node': 'al
 client = ApplicationNode()
 
 
-def assert_basic_application():
+def assert_basic_application(search_in_file):
     resp = client.get()
     assert resp['headers']['Content-Type'] == 'text/plain', 'basic header'
     assert resp['body'] == 'Hello World\n', 'basic body'
 
+    assert search_in_file(r'constructor was called with unsupported') is None
 
-def test_node_application_basic():
+
+def test_node_application_basic(search_in_file):
     client.load('basic')
 
-    assert_basic_application()
+    assert_basic_application(search_in_file)
 
-def test_node_application_options():
+
+def test_node_application_options(search_in_file):
     client.load('options')
 
-    assert_basic_application()
+    assert_basic_application(search_in_file)
 
 
-def test_node_application_loader_unit_http():
+def test_node_application_loader_unit_http(search_in_file):
     client.load('loader/unit_http')
 
-    assert_basic_application()
+    assert_basic_application(search_in_file)
 
 
-def test_node_application_loader_transitive_dependency():
+def test_node_application_loader_transitive_dependency(search_in_file):
```